pick.lic

```diff
@@ -90,8 +90,8 @@ class Pick
     @all_trap_messages = picking_data['traps']
     @disarmed_trap_messages = picking_data['disarmed_traps']
 
-    @harvest_traps = @settings.harvest_traps
-    @component_container = @settings.component_container
+    @harvest_traps = @settings.pick['harvest_traps'] || @settings.harvest_traps
```

Comment:
Should these two be added to base.yaml? https://github.com/rpherbig/dr-scripts/blob/master/profiles/base.yaml#L962

Reply:
Added.